frontend/assets/stylesheets/objects/_grid.scss

```diff
@@ -113,10 +124,55 @@
     }
 }
 
-/* 3up Stacked (3 > 1)
+/* 3up Stacked (Cols per row: 3 on larger viewport > 1 on smaller)
    ========================================================================== */
 
+// 3 on desktop > 1 on tablet > 1 on mobile
 .grid--3up-stacked {
+    &.grid--equal-heights {
+        @include mq(desktop) {
+            display: flex;
+        }
+        .grid__item {
+            @include mq(desktop) {
+                display: flex;
```

Comment:
Couple of things about the flexbox use:

1. Mind posting a couple of screenshots from non-flexbox browsers
2. Probably worth a check in Firefox and Safari as flexbox is occasionally buggy using this shorthand version.
3. It might be worth wrapping this code in a check so it's excluded from our old-IE stylesheet:

```
@if ($browser-supports-flexbox) {
            // flex styles
        } @else {
            // non-flex styles
        }
```